admiral/envs/components/health.py

```diff
@@ -29,41 +29,6 @@ def __init__(self, min_health=0.0, max_health=1.0, initial_health=None, **kwargs
         self.max_health = max_health
```

Comment:
There is not a good enough distinction between health and life agents to merit two classes. I never use health without life. If that use case arises, then we can break it down. But for now, let's merge them together into one to simplify.